### PR TITLE
Bug Fix: Webhook secret stored incorrectly in state

### PR DESCRIPTION
**Pull request by megan07**
Addressing #171 where the webhook secret set in Terraform makes signatures not match.
Before fix:
```
--- FAIL: TestAccGithubRepositoryWebhook_importSecret (9.38s)
    testing.go:568: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

        (map[string]string) (len=1) {
         (string) (len=22) "configuration.0.secret": (string) (len=8) "********"
        }


        (map[string]string) (len=1) {
         (string) (len=22) "configuration.0.secret": (string) (len=18) "RandomSecretString"
        }
```
After fix:
```
=== RUN   TestAccGithubRepositoryWebhook_importSecret
=== PAUSE TestAccGithubRepositoryWebhook_importSecret
=== CONT  TestAccGithubRepositoryWebhook_importSecret
--- PASS: TestAccGithubRepositoryWebhook_importSecret (9.66s)
PASS
```

### CODE REVIEW

File: github/resource_github_organization_webhook.go [hunk 1]
```diff
@@ -92,6 +92,11 @@ func resourceGithubOrganizationWebhookCreate(d *schema.ResourceData, meta interf
 	}
 	d.SetId(strconv.FormatInt(*hook.ID, 10))
 
+	if hook.Config["secret"] != nil {
+		hook.Config["secret"] = webhookObj.Config["secret"]
```
Comment (paddycarver) [inline github/resource_github_organization_webhook.go]: I'm assuming the GitHub API doesn't return the secret, so we have to set it based on what's in the ResourceData? If so, can we add a comment saying as much, just to explain why we have to do this?

Comment (paddycarver) [reply github/resource_github_organization_webhook.go]: Also, what does `hook.Config` get set to, if not `nil`? That looks like the GitHub response, but what does it actually get set to?

File: github/resource_github_organization_webhook.go [hunk 2]
```diff
@@ -134,6 +139,15 @@ func resourceGithubOrganizationWebhookRead(d *schema.ResourceData, meta interfac
 	d.Set("url", hook.URL)
 	d.Set("active", hook.Active)
 	d.Set("events", hook.Events)
+
+	if len(d.Get("configuration").([]interface{})) > 0 {
+		currentSecret := d.Get("configuration").([]interface{})[0].(map[string]interface{})["secret"]
+
+		if hook.Config["secret"] != nil {
+			hook.Config["secret"] = currentSecret
```
Comment (paddycarver) [inline github/resource_github_organization_webhook.go]: Likewise, a comment here just explaining the situation would be 💯

**Comment by harlowja** [comment]
Ah, this explains why terraform screwed up all of our organizations hook secrets! Can we get this merged....

**Comment by zhenkaiwang** [comment]
Hope this get reviewed and merged soon :rocket:

**Comment by wenbin1989** [comment]
Hope this get merged ASAP. we have to apply the patch manually currently, which is not very nice.

**Pull request merged**